server/deskroom/categories/endpoints.py

```diff
@@ -13,28 +13,16 @@
 
 @router.put("/", response_model=UpdateCategoriesOut)
 async def update_categories(
-    update_request: UpdateCategoriesIn,
+    request: UpdateCategoriesIn,
     supabase: AsyncClient = Depends(create_supabase_async_client),
     current_user: DeskroomUser = Depends(get_current_user),
 ) -> UpdateCategoriesOut:
-    id = update_request.id
-    name = update_request.name
-    description = update_request.description
-    org_key = update_request.org_key
+    # delete None values
+    data = {k: v for k, v in request.dict().items() if v is not None}
 
     # if id is None, create new category
-    if id is None:
-        response = (
-            await supabase.table("knowledge_categories")
-            .insert(
-                {
-                    "name": name,
-                    "description": description,
-                    "org_key": org_key,
-                }
-            )
-            .execute()
-        )
+    if request.id is None:
+        response = await supabase.table("knowledge_categories").insert(data).execute()
```

Comment:
```suggestion
        response = await supabase.table("knowledge_categories").insert(request.dict(exclude_none=True)).execute()
```
요렇게 하면 없는 것들을 뺀 체로 나올거에요. 
https://docs.pydantic.dev/1.10/usage/exporting_models/#modeldict